ui/raidboss/emulator/data/network_log_converter/LineEvent.ts

```diff
@@ -1,45 +1,50 @@
 import EmulatorCommon from '../../EmulatorCommon';
+import LogRepository from './LogRepository';
 
 /**
  * Generic class to track an FFXIV log line
  */
 export default class LineEvent {
-  constructor(repo, line, parts) {
-    this.offset = 0; // To be calculated later
-
-    this.parts = parts;
-
-    this.decEvent = parseInt(this.parts[0]);
+  public offset = 0;
+  decEvent: number;
+  hexEvent: string;
+  timestamp: number;
+  networkLine: string;
+  convertedLine = '';
+  invalid = false;
+  index = 0;
```

Comment:
Are these all public? Some private?

Reply:
They're all public. I used vscode's auto-generate-members option to add them, I'm not sure why it added a public for offset but not for any of the others. Should they all have the public modifier since they're all intended to be public?

Reply:
I think consistency is the most important thing at the moment.  I might consider adding https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-member-accessibility.md at some point, and if I do, then everything can get fixed up then.

Reply:
Adjusted to all `public`. I think in this case it makes sense to be more explicit.